code/datums/mob_stats.dm

```diff
@@ -20,6 +33,34 @@
 
 
 
+
+//Some helper procs for mobs, just wrappers to make things easier
+/mob/proc/getStat(statName, Pure = null)
+	if (!istype(src, /mob/living))
+		return 0
+
+	var/mob/living/L = src
+	if (!L.stats)
+		return 0
+
+	return L.stats.getStat(statName, Pure)
+
+
+/mob/proc/changeStat(statName, Pure = null)
+	if (!istype(src, /mob/living))
+		return 0
+
+	var/mob/living/L = src
+	if (!L.stats)
+		return 0
+
+	return L.stats.changeStat(statName, Pure)
+
```

Comment:
Why that is `/mob` not `/mob/living` procs?

Reply:
Ease of use. This is a standard with almost all mob procs. It's how most of the inventory and damage stuff is handled

You define it at the /mob level but make it do nothing. Then you override it at the levels you care about and add features there.
It allows coding to be done with less typecasting

Reply:
Infact, this should probably be written into our coding standards. It's how these things are done because it's a good method.

Mob procs should almost always be defined at the /mob level and overridden later

Reply:
Do we really need to give all this things to `/mob/` instead of `/mob/living/` ?
I mean if only humans and noone can use this - this should really be defined only for `/mob/living/` to not to rubbish already dirty mob's codebase.

Reply:
yes, we should, see above. This is established standard for all mob related procs.

Reply:
but its been the standard since before eris was established, this is how bay designed all mob procs.
If you want to change it enmasse in future we can discuss that. but no reason for an exception right now